processccd.rst

```diff
@@ -4,56 +4,59 @@
 ProcessCcdTask
 ##############
 
-ProcessCcd (which is a ``Cmd Line Task``, and thus can be run directly
-by typing processCcd.py at a shell prompt) executes the steps of how
-an image is processed from raw data to a science-grade images and catalogs that can
-be used in analyses.
+ProcessCcdTask (available as the ``processCcd.py`` ``command line
```

Comment:
I split this into 2 sentences now -- felt it was unwieldy with so many parentheticals.  Input welcome.